src/chainparams.cpp

```diff
@@ -249,13 +254,95 @@ class CTestNetParams : public CChainParams {
     }
 };
 
+/**
+ * Signet
+ */
+class SigNetParams : public CChainParams {
+public:
+    explicit SigNetParams(const ArgsManager& args) {
+        std::vector<uint8_t> bin;
+        vSeeds.clear();
+
+        if (!args.IsArgSet("-signet_blockscript")) {
+            LogPrintf("Using default signet network\n");
+            bin = ParseHex("512103ad5e0edad18cb1f0fc0d28a3d4f1f3e445640337489abb10404f2d1e086be43051ae");
```

Comment:
`"OP_1 03ad5e0edad18cb1f0fc0d28a3d4f1f3e445640337489abb10404f2d1e086be430 OP_1 OP_CHECKMULTISIG`

Why did you choose 1-of-1 multisig instead of a single `OP_CHECKSIG` ?

Reply:
Mostly as I expect there to be backup signers and such, so having it on a multisig setup from the start seems sensible.